packages/accounts-password/password_client.js

```diff
@@ -215,6 +215,7 @@ Accounts.changePassword = (oldPassword, newPassword, callback) => {
  * @locus Client
  * @param {Object} options
  * @param {String} options.email The email address to send a password reset link.
+ * @param {Boolean} options.detailedErrorFlag optional parameter. This determines how detail the error message returned from the server should be.
```

Comment:
Would be nice to have a bit more info here. This will show up in documentation, so it should include that by default it is false and that people should keep it that way for security purposes.